src/renderer.tsx

```diff
@@ -34,6 +34,7 @@ import React from 'react'
 import { render } from 'react-dom'
 import { hot } from 'react-hot-loader' // has to stay first
 import { Provider } from 'react-redux'
+import log from 'electron-log'
```

Comment:
I suggest to assign needed log to console:
```
console.log = log.log
console.error = log.error
```
Because in javascript we get used to console.log, it's faster to use it rather then importing log

Reply:
Fixed